src/components/TopButtons.tsx

```diff
@@ -78,6 +79,10 @@ export const TopButtons = ({ selector, hideSequenceTableButton = false }: Props)
     );
   }
 
+  if (hideTaxonium) {
```

Comment:
This could easily lead to a mistake if we change the order of the buttons or add a button after the Taxonium button. It would be safer to only insert the button if `hideTaxonium` is `false`.